app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -252,6 +258,18 @@ export default function createRPCMethodTrackingMiddleware({
         }
         ///: END:ONLY_INCLUDE_IF
 
+        const isConfirmationRedesign =
+          isConfirmationRedesignEnabled &&
+          (REDESIGN_APPROVAL_TYPES.find((type) => type === method) ||
```

Comment:
Should we simplify this to:

`[...REDESIGN_APPROVAL_TYPES, ...REDESIGN_TRANSACTION_TYPES].find(type => type === method)`

Reply:
absolutely! ty [3531395](https://github.com/MetaMask/metamask-extension/pull/24182/commits/3531395431a7824894ca06ce3b5e094bb38bc979)